src/app/views/sessions/index.html.erb

```diff
@@ -1,7 +1,7 @@
 <% title('All Sessions') %>
 (most recently added first)
 
-<% cache @sessions do %>
+<% cache [@sessions, current_participant] do %>
```

Comment:
The comment’s just going to say “fragment contains user-specific info,” which I feel like the cache key itself already conveys. But I’ll add it.